src/routes/api.js

```diff
@@ -0,0 +1,130 @@
+'use strict';
+
+import express from 'express';
+import modelFinder from '../middleware/modelFinder';
+import auth from '../auth/middleware';
+import Team from '../models/team';
+import User from '../auth/user';
+const router = express.Router();
+
+router.param('model', modelFinder);
+
+/***********************************
+*     POST REQUESTS                *
+************************************/
+router.post('/team', auth, (req, res, next) => {
+
+  if (req.user.role === 'coach') {
+    req.body.coach = req.user._id;
+
+    let document = new Team(req.body);
+
+    document.save()
+      .then(data => {
+        res.send(data);
+      })
+      .catch(next);
+
+  } else {
+    res.status(401);
+    res.send('only coaches may create teams');
+  }
+
+});
+
+router.post('/:model', auth, (req, res, next) => {
+
+  req.body.user = req.user._id;
+
+  let document = new req.model(req.body);
+
+  document.save()
+    .then(data => {
+      res.send(data);
+    })
+    .catch(next);
+});
+
+/***********************************
+*     GET REQUESTS                 *
+************************************/
+router.get('/hello', (req, res, next) => {
```

Comment:
Ok to add a 'hello world' route when first starting development on your project, but eventually it needs to be removed before deploying to production.